doc/index.rst

```diff
@@ -53,6 +52,8 @@ Documentation
    plotting
    api
    internals
+   contributing
+   whats-new
```

Comment:
I noticed that this change moved `whats-new` to the end. I'm not sure this is a good idea.

It's true that the details of whats-new is unlikely to be relevant to new users. But I like putting it first because I think it's something most users will want to see:
1. New users will check it to see if the project is well-maintained.
2. Existing users want to know what's new.

Reply:
That said... Google analytics shows "What's new" ranking pretty lowly in popularity:
https://docs.google.com/spreadsheets/d/1F6IZ0aSQlX7yZDZbS2R9RZN9PAt70Dl_g60BM5agDIc/edit?usp=sharing

Reply:
Still, I think your argument is valid. People won't return to a "what's new" page as often as they do for the other docs, but the page shows that things happen on the project

Reply:
I think my thinking was that new users are probably best served by getting to the why xarray/tutorial/examples pages before. I'm fine moving it back if we think it is worth it.